test/database/int.spec.ts

```diff
@@ -2809,6 +2809,16 @@ describe('database', () => {
     }
   })
 
+  it('should update simple', async () => {
+    const post = await payload.create({ collection: 'posts', data: { title: 'hello' } })
```

Comment:
Can you add another field to the initial data here? E.g `text: 'text'`

That way, when we call `db.updateOne` with just `title`, we can check if `res.text` still exists => this would test if the optimization is actually called by making sure the entire row is not nuked